### DISCUSSION TITLE
AniList Authorization won't work

### DISCUSSION THREAD

**Discussion by Batorian** [vosmiic/jellyfin-ani-sync#151] [Q&A]
I have followed the instructions as written in the wiki. First I create an App in AniList with `http://aaa.bbb.ccc.dd:eeee/AniSync/authCallback` as the Redirect URL. After that I test it via `http://aaa.bbb.ccc.dd:eeee/AniSync/apiUrlTest`, which returns a blank site, so I am assuming it works, as it does not return any kind of error. After that I use my Client ID to Authorize the User. I get the url `https://anilist.co/api/v2/oauth/authorize?response_type=code&client_id={ClientID}&redirect_uri=http://::ffff:aaa.bbb.ccc.dd:eeee/AniSync/authCallback` and remove the `::ffff:` prefix to receive `https://anilist.co/api/v2/oauth/authorize?response_type=code&client_id={ClientID}&redirect_uri=http://aaa.bbb.ccc.dd:eeee/AniSync/authCallback` as url. I open it and login to AniList and click the Authorize button and get redirected to a new site with the following message:
![image](https://github.com/user-attachments/assets/cc2c0fbb-c43b-42a6-90c2-6c69d532b302)
I have not been able to fix this issue after trying out multiple other urls, using private browser etc. I'd appreciate some help.

**Comment by vosmiic** [comment]
Is an error added to the Jellyfin logs when you attempt authentication?
